### ISSUE TITLE
Customer creates a new account via customer/account/create/ -> optin email?

### ISSUE THREAD

**Issue by seansan** [mailchimp/mc-magento#358]
Customer creates a new account via customer/account/create/ -> optin email?

If a customer creates an account via customer/account/create/ 
there is a checkmark "subscribe"

if I check this: am I subscribed immediately + I dont have to optin again: I am subscribed?

if I dont check this: then I am not subscribed. But does this trigger the OPTIN email from MC? Because we have never set and used the OPTIN confirmation email ... but suddenly we see customers that receive this email. 

(if this is the case)
Is this something we need a setting for in the backend?

![image](https://user-images.githubusercontent.com/652395/27830167-6e743506-60c5-11e7-8035-810b59c51616.png)

**Comment by Santiagoebizmarts** [comment]
Hello @seansan 

This is not really an issue. When someone subscribes while opt-in disabled the extension sends the request, if it fails due to that subscriber being added to several lists lately then it subscribes the customer again but sending the confirmation email to confirm the subscription is really wanted by the customer.
We find a problem that caused subscribers to be sent several times triggering this error but it is fixed now and will be ready in next release.

In other words, updating the module to the new version that is going to be released next week should fix your problem.

Kind Regards.
